editor.planx.uk/src/@planx/components/shared/Preview/MoreInfo.tsx

```diff
@@ -59,6 +62,8 @@ const CloseButton = styled("div")(({ theme }) => ({
   color: theme.palette.text.primary,
 }));
 
+const isUsingFeatureFlag = () => hasFeatureFlag("SHOW_INTERNAL_FEEDBACK");
```

Comment:
I _think_ we'd save a few renders by just making this a static value?

`const isUsingFeatureFlag = hasFeatureFlag("SHOW_INTERNAL_FEEDBACK");`

Reply:
That's an interesting one, I think I just copied the pattern from the other places where a feature flag is used and didn't stop to think about it 😅 

I'll experiment with making it static to see whether it's like this for a reason or maybe just causing unnecessary renders.

Reply:
Experimenting with this it seems the static version is grand, when the flag is toggled it triggers the refresh so it seems to be working as intended. 

Fixed: adcb8c2c691d9c78a0655d1c0c7aaf7c3b98e189